sda-commons-server-dropwizard/src/main/java/org/sdase/commons/server/dropwizard/metadata/DetachedMetadataContext.java

```diff
@@ -0,0 +1,34 @@
+package org.sdase.commons.server.dropwizard.metadata;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+
+/**
+ * A {@link MetadataContext} representation which internals <strong>do not reflect changes</strong>
```

Comment:
I don't understand the word "internals" here. Would also be "does not" grammatically.